kinematics_base/include/moveit/kinematics_base/kinematics_base.h

```diff
@@ -56,6 +56,48 @@ class RobotState;
 namespace kinematics
 {
 
+/*
+ * @enum DiscretizationMethods
+ *
+ * @brief Flags for choosing the type discretization method applied on the redundant joints during an ik query
+ */
+namespace DiscretizationMethods
```

Comment:
As far as namespaces to encapsulate enumerations I followed what's described [here](http://wiki.ros.org/CppStyleGuide#Enumerations), should I follow a different convention?